code/__DEFINES/traits/mob.dm

```diff
@@ -90,3 +90,7 @@ DATUM_TRAIT(/mob, TRAIT_DEAF)
 // This mob can breathe water
 #define TRAIT_MOB_WATER_BREATHER		"mob_water_breather"
 DATUM_TRAIT(/mob, TRAIT_MOB_WATER_BREATHER)
+
+// Mob AI interaction Traits
+#define TRAIT_IGNORED_BY_AI				"Ignored by AI"
```

Comment:
`mob_ai_ignore`
do `TRAIT_MOB_AI_IGNORED`
namespacing is important so we can tell at a glance it should never be on something that isn't a mob